tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -7869,15 +7882,46 @@ TEST_F(SQLiteFixture, OutputReportTabular_EndUseBySubcategorySQL)
         ASSERT_EQ(2u, result.size()) << "Failed for query: " << query;
     }
 
-    // Get all subcat usage for all fuels (6)
+    // Get all subcat usage for all fuels (13)
     {
         std::string query("SELECT Value From TabularDataWithStrings"
                 "  WHERE TableName = 'End Uses By Subcategory'"
                 "  AND ReportName = 'AnnualBuildingUtilityPerformanceSummary'"
                 "  AND RowName = 'Exterior Lighting:AnotherEndUseSubCat'");
         auto result = queryResult(query, "TabularDataWithStrings");
 
-        ASSERT_EQ(6u, result.size()) << "Failed for query: " << query;
+        ASSERT_EQ(13u, result.size()) << "Failed for query: " << query;
+    }
+
+    // check the result size for all fuels (including disaggregated "additional fuel")
+    {
+        std::string query("SELECT Value From TabularDataWithStrings"
```

Comment:
I don't like these SQL queries... but that's nor your fault.

It looks like you're looking for __electric__ categories. Should we be looking for fuel categories?

Reply:
I tried to use fuel categories instead of electricity, but there was a crash. It seemed like the existing unit test uses a function to create an object for Electricity with a function (https://github.com/NREL/EnergyPlus/blob/develop/tst/EnergyPlus/unit/OutputReportTabular.unit.cc#L7734) to avoid a crash, but I couldn't find the functions like this for the other fuel yet. I'll figure out it more and try to change it to the fuel category. If I have questions about this, I'll let you know. Thank you for all your reviews! :)